README.md

```diff
@@ -1,9 +1,41 @@
-# Hetu
+# Chronos
 
-This is the developing prototype of Hetu, a general-propose message-passing framework which consider causality as first class citizen.
```

Comment:
Describe the purpose of the project, what capabilities it provides externally, and provide two use case scenarios